python/python/tests/test_async_client.py

```diff
@@ -5072,6 +5073,86 @@ async def endless_xread_call():
         with pytest.raises(RequestError):
             await redis_client.xread({key1: stream_id1, string_key: stream_id1})
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_xgroup_create_xgroup_destroy(
+        self, redis_client: TRedisClient, cluster_mode, protocol, request
+    ):
+        key = get_random_string(10)
```

Comment:
In XREAD, you use key1 = f"{{testKey}}:1-{get_random_string(10)}".

Reply:
I do find prefixing the keys with something unique to the test is helpful for debugging. 
Same for group_name, etc.

Reply:
Yeah, that's because XREAD is a multi-key command, so the keys need to have the same hash tag in order to be mapped to the same hash slot. XGROUP CREATE/DESTROY are single-key commands so they do not have this requirement.